### ISSUE TITLE
Villager Summon Golem crash

### ISSUE THREAD

**Issue by skyjay1** [MinecraftModDevelopment/Extra-Golems#38]
Reported by AcidGamesStreams:
> [https://pastebin.com/8kVMaFcP](https://pastebin.com/8kVMaFcP)
> So i am playing around with some of the 1.14 mods new and old ones and got a crash and it mentions golems but also villagers so i just wanted to see if you could figure out what was the cause and if it was your mod causing it or just a bug overall with something else. :)

**Comment by skyjay1** [comment]
Temporary fix: Open up the config file and disable this behavior by setting the following value to zero:
```
#Percent chance for a villager to successfully summon an Extra Golems golem
#Range: 0 ~ 100
villager_summon_chance = 0
```

**Comment by WenXin20** [comment]
Hello, I got the same crash in 1.14, but I can't find the config file. What's the name of the file?

**Comment by skyjay1** [comment]
In 1.14, config files are now saved per-world. To find the config for this mod go to your .minecraft folder (if using Windows, go to %appdata% to get there). Click on "saves", select the name of the world, and go into "serverconfig". There you will find a file named "golems-server.toml" which is the config file for this mod. Hope that helps

**Comment by WenXin20** [comment]
Thanks! I was able to find the config file location.

**Comment by skyjay1** [comment]
Hopefully fixed in release 9.0.4. If it happens again, let me know
